content/docs/command-reference/remove.md

```diff
@@ -101,7 +101,38 @@ deleted (just the `model` file in this example):
 ```dvc
 $ dvc remove train --outs
 $ ls
-dvc.lock  dvc.yaml  foo.csv  foo.csv.dvc  train.py
+dvc.lock  dvc.yaml  data.csv  data.csv.dvc  train.py
 ```
 
 > Notice that the dependencies (`data.csv` and `train.py`) are not deleted.
+
+## Example: remove a single output file by name
+
+Imagine the same <abbr>workspace</abbr> as before but now we have multiple
+`outs` for the `train` stage:
```

Comment:
```suggestion
## Example: remove a stage output by filename

Imagine the we have multiple <abbr>outputs</abbr> for a `train` stage:
```

Reply:
But why focus the example on stage outputs? What about making it about tracked data in general? Showing both removing an added dataset (dir), as well as a stage output (a model), for example. And/or update a previous example to mention the alternative to targeting .dvc files.

Reply:
You are totally right, I was too focused on stage outputs because that was the "use case" described in the issue. 

And now that I think about it, does it make sense to document targeting `.dvc` files *with* and *without*  the `.dvc` suffix?. I.e. why would a user prefer `dvc remove foo.csv.dvc` instead of just `dvc remove foo.csv`?

Reply:
Happens. And we can go with this for now (just needs some fixes as indicated by Ivan) and I'll change it later. Unless you don't mind reworking it in a more general form along the way 🙂

Reply:
Documenting this change is being definitely more challenging than implementing it xD. 

@jorgeorpinel After reading https://github.com/iterative/dvc/issues/2575#issuecomment-707868202 , I'm thinking that maybe we should remove the example `dvc remove data.csv.dvc` in favor of `dvc remove data.csv`, given that the behavior is now the same but new users might don't be aware of the concept of `.dvc` files?

Reply:
The existing section `## Example: remove a tracked file` kind of overlaps in content with a the separate page `# How to Stop Tracking Data`.

Reply:
Thanks for the updates @daavoo, I can wrap this up once https://github.com/iterative/dvc/pull/5757 is merged.